### ISSUE TITLE
Attempt to whiteout outside of rootfs error with libpng

### ISSUE THREAD

**Issue by ebridge2** [apptainer/singularity#1592]
## Version of Singularity:

`2.5.1-dist`

### Expected behavior

Singularity container builds successfully from existing docker container pull.

### Actual behavior

```
ebridge@cortex:~$ SINGULARITY_TMPDIR=/tmp singularity pull docker://neurodata/m3r-release
WARNING: pull for Docker Hub is not guaranteed to produce the
WARNING: same image on repeated pull. Use Singularity Registry
WARNING: (shub://) to pull exactly equivalent images.
Docker image path: index.docker.io/neurodata/m3r-release:latest
Cache folder set to /home/ebridge/.singularity/docker
Importing: base Singularity environment
tar: ./.exec: implausibly old time stamp -9223372036854775808
tar: ./.run: implausibly old time stamp -9223372036854775808
tar: ./.shell: implausibly old time stamp -9223372036854775808
tar: ./.singularity.d/actions/exec: implausibly old time stamp -9223372036854775808
tar: ./.singularity.d/actions/run: implausibly old time stamp -9223372036854775808
tar: ./.singularity.d/actions/shell: implausibly old time stamp -9223372036854775808
tar: ./.singularity.d/actions/start: implausibly old time stamp -9223372036854775808
tar: ./.singularity.d/actions/test: implausibly old time stamp -9223372036854775808
tar: ./.singularity.d/actions: implausibly old time stamp -9223372036854775808
tar: ./.singularity.d/env/01-base.sh: implausibly old time stamp -9223372036854775808
tar: ./.singularity.d/env/90-environment.sh: implausibly old time stamp -9223372036854775808
tar: ./.singularity.d/env/95-apps.sh: implausibly old time stamp -9223372036854775808
tar: ./.singularity.d/env/99-base.sh: implausibly old time stamp -9223372036854775808
tar: ./.singularity.d/env/99-runtimevars.sh: implausibly old time stamp -9223372036854775808
tar: ./.singularity.d/env: implausibly old time stamp -9223372036854775808
tar: ./.singularity.d/libs: implausibly old time stamp -9223372036854775808
tar: ./.singularity.d/runscript: implausibly old time stamp -9223372036854775808
tar: ./.singularity.d/startscript: implausibly old time stamp -9223372036854775808
tar: ./.singularity.d: implausibly old time stamp -9223372036854775808
tar: ./.test: implausibly old time stamp -9223372036854775808
tar: ./dev: implausibly old time stamp -9223372036854775808
tar: ./environment: implausibly old time stamp -9223372036854775808
tar: ./etc/hosts: implausibly old time stamp -9223372036854775808
tar: ./etc/resolv.conf: implausibly old time stamp -9223372036854775808
tar: ./etc: implausibly old time stamp -9223372036854775808
tar: ./home: implausibly old time stamp -9223372036854775808
tar: ./proc: implausibly old time stamp -9223372036854775808
tar: ./root: implausibly old time stamp -9223372036854775808
tar: ./singularity: implausibly old time stamp -9223372036854775808
tar: ./sys: implausibly old time stamp -9223372036854775808
tar: ./tmp: implausibly old time stamp -9223372036854775808
tar: ./var/tmp: implausibly old time stamp -9223372036854775808
tar: ./var: implausibly old time stamp -9223372036854775808
tar: .: implausibly old time stamp -9223372036854775808
Exploding layer: sha256:ce640ed7800c29984dfd6c4cda9e5b6c759f630356e972c851125713628cdfd9.tar.gz
Exploding layer: sha256:1507ad3dd17ec10052abacf32495c96c09d2cee65f00ffaf04279a589414dcb2.tar.gz
Exploding layer: sha256:ee804876babe30c0e07b6b5e2c619edb7a2fb2faf36037e4bbf82275dd2b2ba8.tar.gz
Exploding layer: sha256:225c66a863d8abfab3c27dbb240caba2b02c92e5b2f88a9b736b40a24dd0f683.tar.gz
Exploding layer: sha256:2df5bb5034a354e48188553e2ae346cca291ba43558ee374888a55284d9b412b.tar.gz
Exploding layer: sha256:4818de87e4e7fb4d060db6be33068bdc1cb8fa71838c8143cbc00e02ea9ad38e.tar.gz
Exploding layer: sha256:2716a7f2dd39e085865ad2815efc9d5e19289b3c6b5b66bf497514eb2332d7e4.tar.gz
Exploding layer: sha256:3af7ec253404ed0724a7c00194e68a6a3a01c65531736cc1d416c1dddead1366.tar.gz
Exploding layer: sha256:3ada869ce28fc5ed20995bb38999bc3474ceb44e505f9554560bd5b648983b20.tar.gz
Exploding layer: sha256:fd27a4540f44943890557e9780f131c837329bff5e7fd11783fca850d006e6bb.tar.gz
Exploding layer: sha256:a67254b0be125ed20045d7fdbcdfa7665734ab921811e45af792c923fbca91df.tar.gz
Exploding layer: sha256:d5df9debada855992e8dfb3848df4901d9d0879553f53b9f47bea09dfbec2bb5.tar.gz
Exploding layer: sha256:099dbc7eb85d0ca46891855900c2ea0dc444f8121f8fc980ab5a77f7afe3f1c2.tar.gz
Exploding layer: sha256:d4714a769ffba9c62825e71b848c8aa24806531ac83e1ce8afc579bf9014a3ff.tar.gz
Exploding layer: sha256:7494b5eedb0be61ad7a4b9cad89c24eae3211e86d4403c53106d7f10865aa72a.tar.gz
Exploding layer: sha256:050918ed2878785de66c76e2e0935d1edfc500fb23caedb7c309718373619c2c.tar.gz
WARNING: Warning reading tar header: Ignoring malformed pax extended attribute
WARNING: Warning reading tar header: Ignoring malformed pax extended attribute
Exploding layer: sha256:1f2322c975bff32597cdec23be08e46f09b06c3afa2fe9ad79a1ef4105baf887.tar.gz
Exploding layer: sha256:d0bbbe7a31113dfa2211b7a8bc0af6c19900ed55325c7eebd2bdd8d341b6e05d.tar.gz
Exploding layer: sha256:304481a876c5052214ded866d076d47a66eccf75810b62cf5aca24158b6a097e.tar.gz
Exploding layer: sha256:d579d0acd3fd4885317feb27163420dd9f1a740f3c6efd4200e57c9bbae7510d.tar.gz
Exploding layer: sha256:bec4481c93281294c7c6ca80ce5b96775433febef25d5901cbcdd91d3f395ec8.tar.gz
Exploding layer: sha256:6d8facf58f35a18def09cf29cdfc609be68b92221a818a1d49b201f02c70c431.tar.gz
ERROR  : Attempt to whiteout outside of rootfs /lib/x86_64-linux-gnu/libpng12.so.0.54.0 - aborting.
ABORT  : Retval = 255
Cleaning up...
ERROR: pulling container failed!
```

### Steps to reproduce behavior

```
singularity pull docker://neurodata/m3r-release:0.0.5
```

Note: the corresponding docker image (which works) has a dockerfile found is [here](https://github.com/ebridge2/FNGS_website/blob/master/cloud/Dockerfile).

**Comment by dtrudg** [comment]
This has the same cause as #1576 - will be working up a PR this week.

**Comment by ebridge2** [comment]
great; thanks! Definitely ping me when pushed! Will this be on a full release branch (with instructions for fixing provided on singularity main site) or would I have to use a development branch in the near future?

**Comment by dtrudg** [comment]
I hope to have it available as a pull request tomorrow, or Monday, and will provide brief instructions of how to test. Once tested and reviewed PRs get merged into the development-2.x branch, and would be brought into the next release when that is done.

**Comment by dtrudg** [comment]
Closing as fixed via #1605  - please reopen if you hit this issue on the latest version of Singularity.
